### PR TITLE
Adds Burrito

### PR DESCRIPTION
**Pull request by Stucey**

### CODE REVIEW

File: Foods.yaml [hunk 1]
```diff
@@ -6,4 +6,4 @@
 
 FavouriteFoods:
   # Add the food to this list on this line, if possible.
-  - ["pizza", "kale_smoothies", "bacon", "cheese", "pie", "carbonara", "garlic_bread", "banana"]
+  - ["pizza", "kale_smoothies", "bacon", "cheese", "pie", "carbonara", "garlic_bread", "banana", "burrito"]
```
Comment (Crafty15) [inline Foods.yaml]: Burritos rule

**Review by Crafty15 — DISMISSED** [review]
good stuff

File: NextTime.yaml [hunk 2]
```diff
@@ -11,5 +11,7 @@ suggestions:
   - Building codebase (B script)
   - CMakeLists
   - Making a module
+  - Bassics of C programming
   - overview of languages
```
Comment (Crafty15) [inline NextTime.yaml:15]: possible duplicate?

**Review by Crafty15 — CHANGES_REQUESTED** [review]

**Review by Crafty15 — APPROVED** [review]

File: Foods.yaml (hunk 1 above)
Comment (Stucey) [reply Foods.yaml]: Sure do

**Pull request merged**